packages/wrangler/src/api/dev.ts

```diff
@@ -69,6 +70,7 @@ export interface UnstableDevOptions {
 export interface UnstableDevWorker {
 	port: number;
 	address: string;
+	proxyData: ProxyData;
```

Comment:
Adding to this public interface so we can fake some startDevWorker events (call the event listeners)

Reply:
TODO add getBindings to this interface